### ISSUE TITLE
Remove online burial form entry points from static pages

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#62552]
## Issue Description
We need to temporarily remove the online form entry point content from these static pages:
- [How to apply for a burial allowance](https://www.va.gov/burials-memorials/veterans-burial-allowance/)
- [Left nav] Remove link

---
## Tasks
- [ ] Draft updates to static pages in drupal
- [ ] Review
- [ ] Send to partners
- [ ] Copyedit and publish

## Acceptance Criteria
- [ ] Entry points are down, pages point to paper form

**Comment by katherine-fung** [comment]
Hi @megzehn @chelsealevinson, 

This page is ready for copy edit. 

The product team aims to launch these changes Wed. 9/27. We'll need to copy edit this page on Monday or Tuesday, and coordinate with Laura Steele and the product team about publishing on Wednesday, 9/27 cc @strelichl  

For copy editing: under the "How do I apply?" section, we: 

- removed the link to the online form 
- added a widget for the info alert telling people the online form no longer works 
- only include the mailing instructions for the application

http://preview-prod.vfs.va.gov/preview?nodeId=864

**Comment by aprocik1** [comment]
Update from the deactivation team: "We confirmed after looking at some sample cases of 530 (burial), we will not deactivate this form. Although it is out of date and there are general quality concerns, sample cases were processed by pension automation as expected."

Reference this [Slack thread](https://dsva.slack.com/archives/C057W3N9K2S/p1698155981409119) for the full update.

TLDR: We do not need to publish these changes. I'm moving this ticket to validate.
